packages/bridge/src/axelar/index.ts

```diff
@@ -604,6 +611,71 @@ export class AxelarBridgeProvider implements BridgeProvider {
 
     return this._assetTransferClient!;
   }
+
+  async getExternalUrl({
+    fromChain,
+    toChain,
+    toAsset,
+    toAddress,
+    fromAsset,
+  }: GetBridgeExternalUrlParams): Promise<BridgeExternalUrl | undefined> {
+    const [axelarChains, axelarAssets] = await Promise.all([
+      getAxelarChains({ env: this.ctx.env }),
```

Comment:
Yeah, I realized that worrying about the bundle size is pointless because, in this PR, I noticed that it's easier and faster to compute the external URLs on the server